lib/game.rb

```diff
@@ -22,9 +23,21 @@ def start
 
   def play
     # use a while loop later to keep game going
-    print_board
-    @board.place_mark(@player_1.mark, @player_1.get_input(@display))
-    print_board
+    while @game_over == false
+      print_board
+      @board.place_mark(@active_player.mark, @active_player.get_input(@display))
+      print_board
+      switch_player
+      @board.place_mark(@active_player.mark, @active_player.get_input(@display))
+      print_board
+      @game_over = true
+    end
+  end
+
+  private
+
+  def switch_player
+    @active_player = @active_player === @player_1 ? @player_2 : @player_1
   end
 
```

Comment:
Do you need this now?  Each player is taking only one turn.  The first place_mark gets the first_player as an argument and the second place_mark get the second_player as an argument.  Remember the bowling game kata - don't get ahead of yourself in terms of design.

Reply:
Yeah I wasn't sure yesterday how many turns I wanted it to run for. I commented everything out.

Reply:
Also be careful about commenting out code.  Commented out code quickly becomes stale.  And, you don't want to lock yourself to a particular design.

Reply:
That's true. But I am reasonably sure I will at least need the switch_players function very soon.